libs/chrome-ext/src/lib/style.css

```diff
@@ -6,6 +6,10 @@ p {
   margin: 0 0 2rem 0;
 }
 
+fieldset {
+  border-style: dashed;
```

Comment:
dashed?
I didn't see the final result, but don't you think a 1px border with a light grey color (like #f0f0f0 or #f7f7f8) would have a better look and feel?